### PR TITLE
oopsy: remove collectSeconds feature

### PR DESCRIPTION
**Pull request by quisquous**
collectSeconds is a lot of implementation complexity
for approximately three triggers.  Just remove it entirely as everything
it does can be done with two triggers.

collectSeconds also used to be used for suppressSeconds, but that's been
implemented now.

This is a breaking change to oopsy user triggers.

### CODE REVIEW

File: ui/oopsyraidsy/data/00-misc/test.js [hunk 1]
```diff
@@ -88,32 +88,44 @@ export default {
...
       netRegexKo: NetRegexes.gameNameLog({ line: '.*나무인형을 쿡쿡 찌릅니다.*?' }),
-      collectSeconds: 5,
-      mistake: (events, data) => {
-        // When collectSeconds is specified, events are passed as an array.
-        const pokes = events.length;
-
+      delaySeconds: 5,
+      mistake: (_e, data) => {
         // 1 poke at a time is fine, but more than one inside of
         // collectSeconds is (OBVIOUSLY) a mistake.
```
Comment (panicstevenson) [inline ui/oopsyraidsy/data/00-misc/test.js:112]: nit: You're removing collectSeconds

File: ui/oopsyraidsy/data/04-sb/raid/o4s.js [hunk 2]
```diff
@@ -115,5 +103,28 @@ export default {
...
+      netRegex: NetRegexes.abilityFull({ id: '241C', ...playerDamageFields }),
+      mistake: (_e, data) => {
+        const arr = data.doubleAttackMatches;
+        if (!arr)
+          return;
+        if (arr.length <= 2)
+          return;
+        // Hard to know who should be in this and who shouldn't, but
+        // it should never hit 3 people.
+        return { type: 'fail', fullText: `${arr[0].ability}  x ${arr.length}` };
```
Comment (panicstevenson) [inline ui/oopsyraidsy/data/04-sb/raid/o4s.js:125]: nit: extra space

**Review by panicstevenson — APPROVED** [review]

File: ui/oopsyraidsy/data/04-sb/raid/o4s.js (hunk 2 above)
Comment (quisquous) [reply ui/oopsyraidsy/data/04-sb/raid/o4s.js:125]: Done.

File: ui/oopsyraidsy/data/00-misc/test.js (hunk 1 above)
Comment (quisquous) [reply ui/oopsyraidsy/data/00-misc/test.js]: Whoops.

**Pull request merged**